src/com/etranslate/tm/processing/rtf/RTFParser.java

```diff
@@ -174,7 +174,7 @@ private static final String getJavaEncoding(int rtfCodepage) {
   // the current number of bytes to skip after a unicode character
   private Integer _currentSkipState = DEFAULT_SKIP_STATE;
   // a stack of skip states for bytes following a unicode character
-  private final Stack _ucSkipStates = new Stack();
+  private final Stack<Integer> _ucSkipStates = new Stack<>();
```

Comment:
This file is generated from RTFParser.jj, so you need to put the fixes in there.